packages/connectivity/src/scp-cf/destination/destination-accessor.ts

```diff
@@ -129,8 +129,8 @@ export async function getAllDestinationsFromDestinationService(
   logger.debug(
     'Attempting to retrieve all destinations from destination service.'
   );
-  if (shouldExchangeToken(options)) {
-    options.jwt = await exchangeToken(options);
+  if (shouldExchangeToken(options) && options.jwt) {
```

Comment:
I assume you only added this so the IDE doesn't complain?
Because shouldExchangeToken only returns true if options.jwt is set.

Reply:
Alternatively you could use `options.jwt!` but I do not really have a [pp] here.

Reply:
exactly. Using `!` is worse in my opinion.